abmarl/examples/sim/__init__.py

```diff
@@ -6,3 +6,6 @@
     MultiAgentContinuousGymSpaceSim, MultiAgentSameSpacesSim
 from .multi_agent_grid_sim import MultiAgentGridSim
 from .reach_the_target import ReachTheTargetSim, RunningAgent, TargetAgent, BarrierAgent
+# TODO: Cannot import traffic corridor because it has a TargetAgent that is not an
```

Comment:
Make a ticket for moving all the imports back to their modules so that we don't conflict.